Breeze-CSS/breeze.css

```diff
@@ -0,0 +1,585 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+
```

Comment:
Bad Practice to use import in your CSS file. Because code below import will execute only after the import call. So a delay on import call can look your page ugly